anomalib/models/draem/loss.py

```diff
@@ -0,0 +1,29 @@
+"""Loss function for the DRAEM model implementation."""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from kornia.losses import FocalLoss, SSIMLoss
+from torch import nn
+
+
+class DraemLoss(nn.Module):
+    """Overall loss function of the DRAEM model.
+
+    The total loss consists of the sum of the L2 loss and Focal loss between the reconstructed image and the input
+    image, and the Structural Similarity loss between the predicted and GT anomaly masks.
+    """
+
+    def __init__(self):
+        super().__init__()
+
+        self.l2_loss = nn.modules.loss.MSELoss()
+        self.focal_loss = FocalLoss(alpha=1, reduction="mean")
+        self.ssim_kornia_loss = SSIMLoss(window_size=11)
```

Comment:
Is it needed to emphasize the kornia in the variable? Why not just `self.ssim_loss`?